src/Builder.php

```diff
@@ -18,6 +18,10 @@
  */
 class Builder
 {
+    // Some configuration options are passed around in an array, using these
+    // keys
+    const CONF_EXPECT_JS_COOKIE  = 'expectJsCookie';
```

Comment:
This configuration is only applicable to the events endpoint. This configuration will be passed to all endpoints. Not a huge problem I guess but seems messy.